packages/vaadin-grid/src/arrayDataProvider.js

```diff
@@ -0,0 +1,142 @@
+/**
```

Comment:
Most of the code in this file is moved from here from [`vaadin-grid-array-data-provider-mixin.js`](https://github.com/vaadin/web-components/blob/a240bccdf50bea736308fe84c7549a4c9b73fb31/packages/vaadin-grid/src/vaadin-grid-array-data-provider-mixin.js)

Reply:
Done